frodo.js

```diff
@@ -1,12 +1,12 @@
-/**
```

Comment:
Nice job on the comprehensive test coverage, it gives a lot of confidence in the stability of the new feature.